03/app.js

```diff
@@ -7,8 +7,56 @@ const buttonSettings = {
     },
     css: {
         border: '1px solid #336699',
-        padding: '5px 20px',
+        padding: '20px 20px',
         color: '#444'
     },
     text: 'Click me!',
-}
\ No newline at end of file
+}
+
+const button = document.createElement("button");
+const section = document.querySelector('.parent-for-button');
+
+for(let key in buttonSettings) {
+    if(key === "attr") {
+        for(let clas in buttonSettings[key]) {
+            if(clas === "className") {
+                button.className = buttonSettings[key][clas];
+                // console.log(buttonSettings[key][clas]);
+            }
+        }
+    }
+}
+
+
+for(let key in buttonSettings) {
```

Comment:
Zwróć uwagę, że pętla zewnętrzna `for ... in` się powtarza - może dałoby się to wykorzystać w taki sposób, aby napisać jedną taką pętlę a nie cztery?